src/main/java/uk/gov/hmcts/reform/pip/account/management/service/MediaApplicationService.java

```diff
@@ -147,19 +147,15 @@ public void deleteApplication(UUID id) {
 
     /**
      * Scheduled job that gets all media applications & sends them to publication services.
+     * Then calls a method to get the applications deleted
      */
     @Scheduled(cron = "${cron.media-application-reporting}")
-    public void processApplications() {
-        processApplicationsForReporting();
-        processApplicationsForDeleting();
-    }
-
-    /**
-     * Collate media applications and send them for reporting.
-     */
     public void processApplicationsForReporting() {
         List<MediaApplication> mediaApplications = getApplications();
-        log.info(publicationService.sendMediaApplicationReportingEmail(mediaApplications));
+        if (!mediaApplications.isEmpty()) {
+            log.info(publicationService.sendMediaApplicationReportingEmail(mediaApplications));
+            processApplicationsForDeleting();
```

Comment:
We've got separate endpoints for deleting / reporting on media applications - those will need to be combined if we are doing deleting / reporting in one method, or the deleting needs to be split out from the above

Reply:
Will combine as it should have always been in one method, it always reports then deletes

Reply:
Updated the controller so there is only one which executes both tasks (will only happen if there are applications to process).

Reply:
Looks good! I've approved this now